ejenti/tasks/hasalTasks.py

```diff
@@ -66,14 +67,77 @@ def checkout_latest_code(**kwargs):
     git_pull(**kwargs)
 
 
+def backfill_hasal_on_perfherder_data(**kwargs):
```

Comment:
base on the `run_hasal_on_latest_nightly()`, should we rename the method name from `backfill_hasal_on_perfherder_data()` to `run_hasal_on_specify_nightly()`?